### DISCUSSION TITLE
Problem processing my student documents.

### DISCUSSION THREAD

**Discussion by seifhamdy** [community/community#42699] [GitHub Education]
I'm a student at UPEI Cairo Campus. When I provided my documentation in my student benefits application, I got an error saying my school doesn't appear to have a campus in Egypt. 

Also, there seems to be a problem with the image processing, because it also says the document I provided, which was a proof of payment, doesn't seem to have my school name, which it does.

What can I do?

**Comment by hiten36** [comment]
Everyone has the opportunity to get this pack and its benefits. Try to edit your github profile - Name, bio, try adding social media links as well (it would help github to identify you), and every field.

There are some discussions on this topic that have been resolved. Here are some links that may help you.

https://github.com/orgs/community/discussions/38918
https://github.com/orgs/community/discussions/30138
https://github.com/orgs/community/discussions/37159
